src/components/accessories/tabs/Tabs.tsx

```diff
@@ -0,0 +1,56 @@
+import classNames from "classnames";
+import React, { Fragment, FunctionComponent, useState } from "react";
+import { IProps } from "./types";
+import "./styles.scss";
+
+const Tabs: FunctionComponent<IProps> = ({
+  header,
+  content,
+}) => {
+
+  const [ currentIndex, updateCurrentIndex ] = useState(0); 
+
+  const renderHeader = (item: object | any, index: number, mobile: boolean = false): JSX.Element => {
+    if(!mobile){
+      return (
+        <div className={classNames("tab", { active: (currentIndex === index) })} key={index} onClick={() => updateCurrentIndex(index)}>
+          <span>{item[index]}</span>
+        </div>
+      );
+    } else {
+      return (
+        <option className={classNames("tab", { active: (currentIndex === index) })} key={index} value={index}>
+          {item[index]}
+        </option>
+      );
+    }
+  };
+
+  const renderContent = (item: object | any, index: number): JSX.Element => {
```

Comment:
I think that better than using `object | any`, it would be a good idea to declare this type as `Record<number, string>`. It's more specific and typescriptworthy.